README.md

```diff
@@ -64,7 +61,7 @@ Instructions assume you are using Puppet Enterprise. For Open Source Puppet inst
   * Click **Review** and then **Submit**.
   * When complete the HEC token should look something like this:
 
-  ![hec_token](https://raw.githubusercontent.com/puppetlabs/puppetlabs-splunk_hec/v0.8.1/docs/images/hec_token.png)
+  ![hec_token](https://raw.githubusercontent.com/puppetlabs/puppetlabs-splunk_hec/main/docs/images/hec_token.png)
```

Comment:
There's a bit of chicken and egg with these urls. We might have to do something like update these urls to refer to the next version tag before we actually create the tag, and then tag a sha with the release version soon thereafter to make the urls work?

Reply:
I updated this to main for instance, but I think I will revert that change and we can deal with this problem later.